src/redismodule.h

```diff
@@ -1511,7 +1511,7 @@ static int RedisModule_Init(RedisModuleCtx *ctx, const char *name, int ver, int
     REDISMODULE_GET_API(BlockedClientMeasureTimeEnd);
     REDISMODULE_GET_API(SetDisconnectCallback);
     REDISMODULE_GET_API(SubscribeToKeyspaceEvents);
-    REDISMODULE_GET_API(AddPostNotificationJob);
+    REDISMODULE_GET_API(AddPostJob);
```

Comment:
not happy with this name.
we're adding a `job` that's a `post <something>`, but it doesn't say post of what.
maybe it should also be a "post action job" or "post modification job", or a "post execution unit job".
@yossigo maybe you have an advise